assets/test/emotableRepository.ts

```diff
@@ -0,0 +1,101 @@
+import { ethers } from 'hardhat';
+import { expect } from 'chai';
+import { BigNumber, Contract } from 'ethers';
+import { loadFixture } from '@nomicfoundation/hardhat-network-helpers';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import { ERC721Mock, EmotableRepository } from '../typechain-types';
+
+function bn(x: number): BigNumber {
+  return BigNumber.from(x);
+}
+
+async function tokenFixture() {
+  const factory = await ethers.getContractFactory('ERC721Mock');
+  const token = await factory.deploy('Chunky', 'CHNK');
+  await token.deployed();
+
+  return token;
+}
+
+async function emotableRepositoryFixture() {
+  const factory = await ethers.getContractFactory('EmotableRepository');
+  const repository = await factory.deploy();
+  await repository.deployed();
+
+  return repository;
+}
+
+describe('RMRKEmotableRepositoryMock', async function () {
+  let token: ERC721Mock;
+  let repository: EmotableRepository;
+  let owner: SignerWithAddress;
+  let addrs: SignerWithAddress[];
+  const tokenId = bn(1);
+  const emoji1 = Buffer.from('😎');
+  const emoji2 = Buffer.from('😁');
+
+  beforeEach(async function () {
+    [owner, ...addrs] = await ethers.getSigners();
+    token = await loadFixture(tokenFixture);
+    repository = await loadFixture(emotableRepositoryFixture);
+  });
+
+  it('can support IEmotableRepository', async function () {
+    expect(await repository.supportsInterface('0x08eb97a6')).to.equal(true);
+  });
+
+  it('can support IERC721', async function () {
+    expect(await repository.supportsInterface('0x80ac58cd')).to.equal(true);
+  });
```

Comment:
Why should it support 721 anymore?

Reply:
Same thing here. This was changed to check for 165 support.